### PR TITLE
test_jenkins: set UCX_IB_ETH_PAUSE_ON=y to allow non-ud testing for an Eth port.

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/2568/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1922/ for details (Mellanox internal link).

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -183,15 +183,15 @@ if [ -n "$JENKINS_RUN_TESTS" ]; then
     mkdir -p $GTEST_REPORT_DIR
 
     echo "Running unit tests"
-    $AFFINITY $TIMEOUT make -C test/gtest test UCS_HANDLE_ERRORS=bt
+    $AFFINITY $TIMEOUT make -C test/gtest test UCS_HANDLE_ERRORS=bt UCX_IB_ETH_PAUSE_ON=y
```
Comment (yosefe) [inline contrib/test_jenkins.sh]: can we somehow test the system on whether RoCE is really lossless?

Comment (yosefe) [inline contrib/test_jenkins.sh]: we need some way to not turn this on if PAUSE is not configured..

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr__NEW/5/ for details.

**Pull request closed without merging**